modules/rosa-sts-aws-requirements-ocm.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa-sts-aws-prereqs.adoc
+
+:_content-type: CONCEPT
+[id="rosa-ocm-requirements_{context}"]
+= Requirements for using {cluster-manager}
+
+The following sections are only required if you use {cluster-manager-url}. If you are only using your CLI tools, then you can disregard these requirements.
+
+To use {cluster-manager}, you must link your AWS accounts. This linking concept is also known as account association.
```

Comment:
Is this addition really a new assembly? Is account association a discrete user story?